.travis.yml

```diff
@@ -201,8 +206,10 @@ before_install:
       conda install pip conda;
       conda update -n base conda;
       conda info -a;
-      conda install pytest pytest-xdist $NUMPYSPEC mpmath Cython pybind11;
+      conda install pytest pytest-xdist numpy mpmath Cython pybind11;
```

Comment:
While unpinning to use the latest numpy release sounds fine, I think this should remain as `$NUMPYSPEC`.